pkg/docker/util.js

```diff
@@ -590,5 +591,55 @@
         return deferred.promise();
     };
 
+    util.find_container_log = function find_container_log(entry, resultCallback) {
+        if ( entry.Elements.indexOf('container_id') < 0 ) {
+            return;
+        }
+
+        entry.ReadElements(['container_id'], 0x4).done(function(data) {
+            if (data.hasOwnProperty('container_id')){
+                // Wait for a while, until ABRT processes the problem and writes it into journal
+                window.setTimeout(util.problem_log, 2000, entry, data['container_id'].v, resultCallback);
```

Comment:
This should poll repeatedly until success for failure, not just once. Processing a large crash can certainly take more than 2s, particularly on a busy system.

Reply:
Yeah, it should. The problem is that, that it may never succeed nor fail. It waits for journal, and when user does not want ABRT to report to journal, he/she can turn it off, which would make this polling forever.